torch/_inductor/triton_heuristics.py

```diff
@@ -492,7 +492,7 @@ def benchmark_all_configs(self, *args, **kwargs):
                     v,
                     k.n_regs,
                     k.n_spills,
-                    k.shared,
+                    k.metadata.shared if hasattr(k.metadata, "shared") else k.shared,
```

Comment:
> So I was a bit coding in the dark. I was unsure which decimal number (%d) is making problems. What was your "roadmap" to solve this problem?

In the error message from the issue, we can see the arguments the logger received:
```python
Message: '%s: %f, nreg %d, nspill %d, #shared-mem %d'
Arguments: (<triton.runtime.autotuner.Config object at 0x7fa33942ead0>, 0.2385919988155365, 31, 8, None)
```

So the `None` was the last argument, which is the shared memory. By comparing against the launch code in the main triton repo, I could also see that the the shared memory argument was the only one that had changed.

Reply:
@peterbell10 OK, I see. Thanks for the explanation.